src/documentation/domains/issue/controller/issue.controller.docs.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * @swagger
+ * components:
+ *   schemas:
```

Comment:
Should change de name of the Entity to TimeTrackingDTO and then should add the class recieved (IssuePauseParams) so the frontend team is aware of validation